src/executables/util_cs.c

```diff
@@ -3581,6 +3581,28 @@ vacuumdb (UTIL_FUNCTION_ARG * arg)
       goto error_exit;
     }
 
+  dump_flag = utility_get_option_bool_value (arg_map, VACUUM_DUMP_S);
+
+  if (dump_flag)
+    {
+      output_file = utility_get_option_string_value (arg_map, VACUUM_OUTPUT_FILE_S, 0);
```

Comment:
if you code this way, users can only use -o with --dump, is it right ?

Reply:
@mhoh3963 You're right. However, I hope to define an error message after defining a full spec for the --dump option. If there is no big problem, can we proceed like that?

Reply:
OK, i got it.